bai-tap-buoi-10/main.js

```diff
@@ -0,0 +1,140 @@
+//Bài 1:
+for(let i = 1 ; i <= 10; i++) {
+  for (let j = 1; j <= 10; j++){
+    let sum = 0
+    sum = i * j;
+    console.log(i + ' * ' + j + ' = ' + sum);
+  }
+}
+
+//Bài 2:
+const getNumberEven = number => {
+  if(number >= 1 && number <= 30){
+    for (let i = 1; i <= number; i++){
+      if (i % 2 == 0){
+        console.log(i)
+      }
+    }
+  } else {
+    console.log('Nhap trong khoang 1-30')
+  }
+}
+
+//Bài 3:
+const getSum = number => {
+  if(number >= 1 && number <= 30){
+    let sum = 0
+    for (let i = 1; i <= number; i++){
```

Comment:
```suggestion
    for (let i = 1; i <= number; i++) {
```
Tham khảo cách bỏ `if else` ở bài trên